From 4198a7a04145ac9946ff9eee0d3bb3c3645f24ef Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Tue, 23 May 2023 11:50:09 -0700
Subject: [PATCH 08/12] Fix clear.cfg

- Use config_opts['use_bootstrap'] instead of
config_opts['use_bootstrap_container'] (deprecated in 2.4.1)
---
 etc/mock/clear.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/mock/clear.cfg b/etc/mock/clear.cfg
index 2065e13..d9a7eba 100644
--- a/etc/mock/clear.cfg
+++ b/etc/mock/clear.cfg
@@ -10,7 +10,7 @@ config_opts['plugin_conf']['root_cache_opts']['extension'] = '.zst'
 config_opts['releasever'] = 'clear'
 config_opts['package_manager'] = 'dnf'
 config_opts['chrootgroup'] = 'mockbuild'
-config_opts['use_bootstrap_container'] = False
+config_opts['use_bootstrap'] = False
 
 config_opts['dnf.conf'] = """
 [main]
-- 
2.42.0

